+++ /usr/share/pve-manager/ext4/pvemanagerlib.js	2013-09-18 14:40:38.000000000 +0200
@@ -446,23 +446,7 @@
 	    },
 	    success: function(response, opts) {
 		var data = response.result.data;
-
-		if (data.status !== 'Active') {
-		    Ext.Msg.show({
-			title: gettext('No valid subscription'),
-			icon: Ext.Msg.WARNING,
-			msg: PVE.Utils.noSubKeyHtml,
-			buttons: Ext.Msg.OK,
-			callback: function(btn) {
-			    if (btn !== 'ok') {
-				return;
-			    }
-			    orig_cmd();
-			}
-		    });
-		} else {
-		    orig_cmd();
-		}
+		orig_cmd();
 	    }
 	});
     },
