--- /usr/share/javascript/proxmox-widget-toolkit/proxmoxlib.js	2019-11-26 13:00:18.000000000 +0100
+++ /usr/share/javascript/proxmox-widget-toolkit/proxmoxlib.js	2019-12-14 11:15:10.207015540 +0100
@@ -378,22 +378,22 @@
 	    success: function(response, opts) {
 		var data = response.result.data;
 
-		if (data.status !== 'Active') {
-		    Ext.Msg.show({
-			title: gettext('No valid subscription'),
-			icon: Ext.Msg.WARNING,
-			message: Proxmox.Utils.getNoSubKeyHtml(data.url),
-			buttons: Ext.Msg.OK,
-			callback: function(btn) {
-			    if (btn !== 'ok') {
-				return;
-			    }
-			    orig_cmd();
-			}
-		    });
-		} else {
+//		if (data.status !== 'Active') {
+//		    Ext.Msg.show({
+//			title: gettext('No valid subscription'),
+//			icon: Ext.Msg.WARNING,
+//			message: Proxmox.Utils.getNoSubKeyHtml(data.url),
+//			buttons: Ext.Msg.OK,
+//			callback: function(btn) {
+//			    if (btn !== 'ok') {
+//				return;
+//			    }
+//			    orig_cmd();
+//			}
+//		    });
+//		} else {
 		    orig_cmd();
-		}
+//		}
 	    }
 	});
     },
